cool-core/src/test/java/com/nus/cool/core/resources/health/table.csv

```diff
@@ -0,0 +1,256 @@
+P-5,1979,labtest,None,None,Labtest-C,20,2012-01-06
```

Comment:
In the latest main branch, We have added the names of columns to CSV files.

Reply:
In the file `fieldtest/table.csv`, the names of columns should be `id,birthYear,event,diagnosis,prescribe,labtest,value,time`

Reply:
the fieldtest/table.csv is the earlier version, which we used to generate TestTable for unittest.
But it can test `HashField` and `RangeField`

I will optimize these dataset and related code later, directly use the the new uploaded three datasets.

Reply:
> these dataset just for test. No need add column header From yaml config file, we can know everything about column header, include field type

the loader has been changed in the latest version. It needs to deal with the name of columns in the table

Reply:
I directly use parser instead of csvloader.